src/ert/scheduler/lsf_driver.py

```diff
@@ -399,5 +431,15 @@ async def _poll_once_by_bhist(self, missing_job_ids: Iterable[str]) -> _Stat:
         self._bhist_cache_timestamp = time.time()
         return _Stat(**{"jobs": jobs})
 
+    def _get_resource_requirement_arg(self) -> List[str]:
```

Comment:
nitpick on function name, this not not a "getter", iti s building a list of strings.